activemq-client/src/main/java/org/apache/activemq/management/StatisticImpl.java

```diff
@@ -83,32 +89,31 @@ public synchronized long getLastSampleTime() {
     /**
      * @return the enabled
      */
-    public boolean isEnabled() {
+    public synchronized boolean isEnabled() {
```

Comment:
There is no benefit to making these methods synchronized, the variables are already volatile and reading or setting the boolean is atomic